### DISCUSSION TITLE
Domain name

### DISCUSSION THREAD

**Discussion by willemml** [willemml/numworks_rust#22] [General]
I am thinking of paying for a domain namme for this project (they are cheap, and it would be neat to have a dedicated website.) The other solution is to just use the nw-rs.github.io domain.

Looking for suggestions on the name.

(Please note that if someone else buys a domain and tries to sell it to me I will not buy it or use it unless it is free and they are paying the transfer fees, so please don't try.)

**Comment by quentinguidee** [comment]
> this is a very undervalued project, and it deserves far more adoption

Note that the fraction of people interested in calculators is extremely small. This can impact the adoption rate considerably, despite the project which is technically very interesting.

(off-topic: If you want it to be more known, I suggest you think about things that could make it adopted by more people. For example, the vast majority of people who have installed Omega are only interested in the game emulator, and not the OS itself. And even with it, the adoption rate is small)

**Reply by willemml** [reply]
Yeah, I have been keeping that in mind, the number of people who have a Numworks isn't tiny, but probably less than 1% are actually interested in custom firmware stuff, and those are only in it for the games... I never expected more than 5-10 people, if even, to use rustworks, it is definitely just a personal project that I enjoy working on every now and then. It being written in rust doesn't help because not many people interested in calculators (probably mostly high school/Lycee students) seem to be interested in Rust.
